src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -173,15 +225,39 @@ const SampleChatInfoScreen = () => {
     );
   };
 
-  const {anon_user_info_color_code, anon_user_info_emoji_code, anon_user_info_emoji_name} =
-    channelInfo?.rawJson?.channel || {};
+  const renderImageComponent = (item) => {
+    if (!isContainUrl(item?.user?.image) || item?.user?.name === ANONYMOUS_USER) {
+      return (
+        <View style={styles.mr7}>
+          <AnonymousIcon
+            color={anon_user_info_color_code}
+            emojiCode={anon_user_info_emoji_code}
+            size={normalize(48)}
+          />
+        </View>
+      );
+    }
+    return (
+      <View style={styles.mr7}>
+        <FastImage style={styles.imageUser} source={{uri: item?.user?.image}} />
+      </View>
+    );
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `renderImageComponent` has been introduced. Make sure it is working as expected and is properly tested.



<!-- This is an auto-generated comment by OSS CodeRabbit -->